components/layout/WidgetPreview/index.js

```diff
@@ -14,14 +13,16 @@ const WidgetPreview = ({
   return (
     <Styled.PreviewPanel $size={size}>
       <Styled.PreviewHeader>
-        <Styled.PreviewTitle>{title}</Styled.PreviewTitle>
-        {openModalCallback && (
-          <Styled.IconButton
-            aria-label={t("summit_dashboard.open_modal")}
-            onClick={() => openModalCallback()}
-          >
-            <IconComposer icon="Plus" />
-          </Styled.IconButton>
+        {openModalCallback ? (
```

Comment:
did some research on best practice for collapsible sections and other icon-header type layouts and decided to re-write both of these into a button nested inside of a heading tag